bin/Powheg/production/pre2017/13TeV/HZJ_JHUGen_HZZ2LX_NNPDF30_13TeV/HZJ_HanythingJ_NNPDF30_13TeV_template.input

```diff
@@ -0,0 +1,71 @@
+vdecaymode   10  ! 1,2,3 => electron,muon,tau; 4,5,6 => respective neutrinos
+hdecaymode  -1  ! undecayed Higgs boson (for PYTHIA and HERWIG)
+
+hmass  {mass}d0        ! Higgs boson mass 
+hwidth {width}d0     ! Higgs boson width 
+
+min_h_mass    10d0      
+max_h_mass  13000d0 
+
+min_z_mass     10d0
+max_z_mass    13000d0
+
+numevts NEVENTS   ! number of events to be generated
+ih1 1             ! hadron 1 (1 for protons, -1 for antiprotons)
+ih2 1             ! hadron 2 (1 for protons, -1 for antiprotons)
+ebeam1 6500d0     ! energy of beam 1
+ebeam2 6500d0     ! energy of beam 2
+
+bornktmin 0.26d0      ! (default 0d0) generation cut. Minimum kt in underlying Born
+bornsuppfact 0d0 ! (default 0d0)  mass param for Born suppression factor. If < 0 suppfact = 1
+
+lhans1  260000         ! pdf set for hadron 1 (LHA numbering)
+lhans2  260000         ! pdf set for hadron 2 (LHA numbering)	
+
+renscfact  1d0   ! (default 1d0) ren scale factor: muren  = muref * renscfact 
+facscfact  1d0   ! (default 1d0) fac scale factor: mufact = muref * facscfact 
+
+! Parameters to allow or not the use of stored data
+use-old-grid    1 ! if 1 use old grid if file pwggrids.dat is present (<> 1 regenerate)
+use-old-ubound  1 ! if 1 use norm of upper bounding function stored in pwgubound.dat, if present; <> 1 regenerate
+
+! the typical call uses 1/1400 seconds (1400 calls per second)
+ncall1 1000000      ! Number of calls for the construction of the importance sampling grid
+itmx1 3           ! Number of iterations for grid
+ncall2 75000      ! Number of calls for the computation of the upper bounding envelope
```

Comment:
to me this looks like a bit low for ncall2, can you please have a look at the top files for the integration grids?
https://twiki.cern.ch/twiki/bin/viewauth/CMS/PowhegBOXPrecompiledCheckGrids